### DISCUSSION TITLE
How to interact with web pages using JavaScript in Dioxus Desktop

### DISCUSSION THREAD

**Discussion by zhtanjj** [DioxusLabs/dioxus#2963] [Ideas]
load  a html from web url, how to call  Dioxus use javascript function in html page. how to call html javascript function in Dioxus .

**Comment by ealmloff** [comment]
You can use the [eval](https://docs.rs/dioxus/0.6.0-alpha.2/dioxus/events/fn.eval.html) method to run arbitrary javascript in dioxus

**Reply by zhtanjj** [reply]
THKS.
yes , can use the  method to run arbitrary javascript in dioxus。
Now I want to exchange data with Dioxus after triggering JavaScript events on a static HTML web page? Are there any sample ?

**Reply by ealmloff** [reply]
The eval example shows how to communicate between javascript and rust with a channel like interface: https://github.com/DioxusLabs/dioxus/blob/main/examples/eval.rs
